dandiapi/api/views/info.py

```diff
@@ -6,8 +6,6 @@
 from rest_framework.decorators import api_view
 from rest_framework.response import Response
 
-import versioneer
```

Comment:
Can you also remove versioneer?

Reply:
If we do that, what should we use for the version in `setup.py`? Should we just pull it out of the `DJANGO_DANDI_API_VERSION` env var?